build.xml

```diff
@@ -4,7 +4,7 @@
  <!-- PROPERTIES -->
  <property file='build-custom.properties' />
  
- <property name='version'   value='20190610.3'/>
+ <property name='version'   value='20190610.4'/>
```

Comment:
Is this change is also a part of upgrading the library and not the Zimbra specific change ?

Reply:
added this change to a separate commit